### ISSUE TITLE
Add support for min/max strings on date input

### ISSUE THREAD

**Issue by andrecasal** [brophdawg11/remix-validity-state#56]
This errors out:

```tsx
type formSchema = {
	inputs: {
		date: InputDefinition
	}
}

const formDefinition: formSchema = {
	inputs: {
		date: {
			validationAttrs: {
				type: 'date',
				min: new Date().toLocaleDateString('en-us') // 👈 TS error
			}
		}
	},
}
```

Error:
```
Type 'Date' is not assignable to type 'number | BuiltInValidationAttrsFunction<number> | undefined'.
```

**Comment by brophdawg11** [comment]
Good catch, this was the first example where a validate attribute value typing differed based on the input `type`: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/min.

This is fixed in 0.10.1.  Also note that you need a `YYYY-MM-DD` format, so you'll need to use:

```js
validationAttrs: {
  type: 'date',
  min: new Date().toISOString().substring(0, 10),
}
```

**Comment by brophdawg11** [comment]
oops, use `0.11.0` instead - there was a new feature in `main` as well
